src/unix/uilocale.cpp

```diff
@@ -594,6 +594,48 @@ wxUILocaleImplUnix::GetInfo(wxLocaleInfo index, wxLocaleCategory cat) const
 
             return GetLangInfo(RADIXCHAR);
 
+        case wxLOCALE_MEASURE_METRIC:
+        {
+            wxString measureStr = GetLangInfo(_NL_MEASUREMENT_MEASUREMENT);
+            if (measureStr.IsEmpty())
+                measureStr = wxString("\0");
```

Comment:
This return value is not documented and it's not clear what does it correspond to.